qanary-component-MT-Python-Helsinki-NLP/boot.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo The port number is: $SERVER_PORT
+echo The Qanary pipeline URL is: $SPRING_BOOT_ADMIN_URL
+if [ -n $SERVER_PORT ]
+then
+    exec gunicorn --timeout 180 -b :$SERVER_PORT --access-logfile - --error-logfile - run:app
```

Comment:
@Perevalov timeout could also be an optional variable. If it does not exist, you can use 180 as a default/fallback.

Reply:
Changed in https://github.com/WDAqua/Qanary-question-answering-components/pull/143/commits/a7c0c9970c1681511eb67aa716321edee8f0364d